src/js/components/Tab/index.d.ts

```diff
@@ -1,10 +1,11 @@
 import * as React from "react";
+import { Omit } from "../../utils";
 
 export interface TabProps {
   plain?: boolean;
-  title?: string | React.ReactNode;
+  title?: React.ReactNode;
```

Comment:
I think it can still be a string `title="Britt"`

Reply:
React.ReactNode covers String element 

React can render a few things like JSX or string. These are all consolidated into the type React.ReactNode so use it for when you want to accept renderables 

Even with the React.ReactNode it will still accept the string 
https://basarat.gitbooks.io/typescript/docs/jsx/react.html

Reply:
My reasoning behind taking out String was because of the fact that ReactNode covers string as one of the types it accepts. After doing my research I found this https://flow.org/en/docs/react/types/?source=post_page---------------------------#toc-react-node
which describes in the docs what ReactNode is and when it is used.